public/index.html

```diff
@@ -39,7 +36,7 @@
 <script defer="defer" src="//ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.4/js/script.js"></script>
 <link rel="stylesheet" href="https://assets.emblstatic.net/vf/v2.4.12/assets/ebi-header-footer/ebi-header-footer.css" type="text/css" media="all" />
 
-<section class="vf-hero vf-hero--1200 | vf-u-fullbleed" style="--vf-hero--bg-image: url(https://acxngcvroo.cloudimg.io/v7/https://www.embl.org/files/wp-content/uploads/EBI_webbanner_test_V4.jpg);">
+<section class="vf-hero vf-hero--1200 | vf-u-fullbleed" style="--vf-hero--bg-image: url(https://www.ebi.ac.uk/metagenomics/proteins/static/img/banner.94dfddd7aebb.jpg);">
```

Comment:
I have localised this image, because the URL on MGnify Proteins is randomly generated on every build so this URL will break very often.